### PR TITLE
#830: Dropdown: on IE11, values in multi-select mode are not vertically aligned when in single line (closes #830)

### LINKED ISSUE #830: Dropdown: on IE11, values in multi-select mode are not vertically aligned when in single line
## description
On IE11, values in multi-select mode are not vertically aligned **when in single line**

image

## how to reproduce
- {optional: describe steps to reproduce bug}

## specs
{optional: describe a possible fix for this bug, if not obvious}

## misc
{optional: other useful info}

### PR DESCRIPTION
**Pull request by sawsanias**
Closes #830

### tests performed
* In IE11, test the default size dropdown:
image
* Add more options till they wrap to next line:
image

*Change the size of the dropdown in the multi example by adding the property `size='small'`, in IE11
image

*Continue adding options till they wrapped to  new line:
image

* Add the property searchable to dropdown and test the dropdown
upvphrcmj1

Repeat the steps above in Chrome, and make sure everything is working.

> These results can be screenshots, query results, or even just pointers to unit tests that successfully passed.

- [x] Google Chrome
- [x] Internet Explorer

> An example of partial test is a field displaying 4 possible values. If 3 values are tested, with screenshots, and 1 is not, then it should be mentioned here.

### CODE REVIEW

File: src/dropdown/dropdown.scss [hunk 1]
```diff
@@ -221,17 +221,10 @@ $height-small: 30px !default;
       height: auto;
 
       .Select-multi-value-wrapper {
-        min-height: $height;
         height: auto;
```
Comment (francescogior) [inline src/dropdown/dropdown.scss]: removing this makes disappear the placeholder

Comment (francescogior) [inline src/dropdown/dropdown.scss]: And the input: if you make it `searchable` you don't see what you type (if there are no value)

I'm not 100% sure, but I think you can fix it using the `.has-value` class and apply the min-height conditionally

File: src/dropdown/dropdown.scss [hunk 2]
```diff
@@ -287,6 +288,8 @@ $height-small: 30px !default;
 
       &.is-small {
         .Select-control .Select-multi-value-wrapper {
+          min-height: auto;
```
Comment (veej) [inline src/dropdown/dropdown.scss]: This looks useless... the previous line seems to include this case too

Comment (francescogior) [inline src/dropdown/dropdown.scss]: I think it is to beat the `.is-small .Select-multi-value-wrapper { min-height: $height-small; }`

**Pull request merged**